index.js

```diff
@@ -163,6 +139,84 @@ function processDuplicateFree(/* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS
 /////////////// HIGHER-ORDER ARRAY METHODS ///////////////
 /////////////// HIGHER-ORDER ARRAY METHODS ///////////////
 
+
+/**
+ * ### Challenge `lowerCaseStrings`
```

Comment:
I believe this function is not being exposed at the bottom of this file, so the test can't pass